portal-web/test/functional/com/liferay/portalweb/macros/asset/assetpublisher/AssetPublisherPortlet.macro

```diff
@@ -327,7 +327,7 @@ definition {
 
 		Navigator.gotoNavTab(navTab = "Display Settings");
 
-		Panel.expandPanel(panel = "Set and Enable");
+		Panel.expandPanel(panel = "set-and-enable");
```

Comment:
Unneeded. This is a regression issue. https://issues.liferay.com/browse/LPS-136032

Reply:
Thanks for letting me know! I will update the PR removing the change.